enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -37,9 +37,13 @@ export const useInitData = ({ config }: InitDataLoaderParameters) => {
   if (api.basename !== config.metabaseInstanceUrl) {
     api.basename = config.metabaseInstanceUrl;
   }
-  if (config.fetchRequestToken !== fetchRefreshTokenFnFromStore) {
-    dispatch(setFetchRefreshTokenFn(config.fetchRequestToken ?? null));
-  }
+
+  useEffect(() => {
+    if (config.fetchRequestToken !== fetchRefreshTokenFnFromStore) {
```

Comment:
I introduced this "error" with a refactor, when I remove the use effect from `api.basename` a few lines above.

Reply:
I didn't realize that could cause an issue when reviewing, very tricky 🤯

Reply:
Yeah it's also hard to notice a new error showing up as we have quite a few of them at the moment :(